resources/js/components/gallery/albumModule/thumbs/PhotoThumb.vue

```diff
@@ -45,14 +45,22 @@
 				hidden: display_thumb_photo_overlay === 'never',
 			}"
 		>
-			<h1 class="min-h-[19px] mt-3 mb-1 ml-3 text-surface-0 text-base font-bold overflow-hidden whitespace-nowrap text-ellipsis">
-				{{ props.photo.title }}
-			</h1>
-			<span v-if="props.photo.preformatted.taken_at" class="block mt-0 mr-0 mb-2 ml-3 text-2xs text-surface-300">
-				<span title="Camera Date"><MiniIcon icon="camera-slr" class="w-2 h-2 m-0 mr-1 fill-neutral-300" /></span
-				>{{ props.photo.preformatted.taken_at }}
-			</span>
-			<span v-else class="block mt-0 mr-0 mb-2 ml-3 text-2xs text-surface-300">{{ props.photo.preformatted.created_at }}</span>
+			<template v-if="photo_thumb_info === 'title'">
+				<h1 class="min-h-[19px] mt-3 mb-1 ml-3 text-surface-0 text-base font-bold overflow-hidden whitespace-nowrap text-ellipsis">
+					{{ props.photo.title }}
+				</h1>
+				<span v-if="props.photo.preformatted.taken_at" class="block mt-0 mr-0 mb-2 ml-3 text-2xs text-surface-300">
+					<span title="Camera Date"><MiniIcon icon="camera-slr" class="w-2 h-2 m-0 mr-1 fill-neutral-300" /></span
+					>{{ props.photo.preformatted.taken_at }}
+				</span>
+				<span v-else class="block mt-0 mr-0 mb-2 ml-3 text-2xs text-surface-300">{{ props.photo.preformatted.created_at }}</span>
+			</template>
+			<template v-else>
+				<h1
+					class="min-h-[19px] mt-3 mb-1 ml-3 text-base overflow-hidden whitespace-nowrap text-ellipsis prose-invert"
+					v-html="props.photo.preformatted.description"
+				></h1>
```

Comment:
```suggestion
				>{{ props.photo.description }}</h1>
```
To show nice ellipses when text overflows maybe?

Reply:
The problem is that in the later case (without v-html) it will no longer be formatted with markdown.

Reply:
Understand. But do we really want to have formatted description text in the photo thumbnail?

Reply:
Looking at first screenshot vs second, I would say it is better. :|

Reply:
![image](https://github.com/user-attachments/assets/e67dca90-5605-4562-af5d-67eb9d76e0ef)
Fixed [2765071](https://github.com/LycheeOrg/Lychee/pull/3263/commits/27650717032cadd140f38843229e666f07ec1740) :)

Reply:
clamp after 3 lines.